index.html

```diff
@@ -11,22 +11,214 @@
+                        </button>
+                </form></div>
+            </div>
+                <br>
+                <hr>
+                <ul class="icons">
+                    <li><a href="www.twitter.com" ><img src="images/twitter.png" alt="twitter"></a></li>
+                    <li><a href="www.linkedin.com"><img src="images/Linkedin.png" alt="Linkedin"></a></li>
+                    <li><a href="#"><img src="images/medium.png" alt="medium"></a></li>
+                    <li><a href="#"><img src="images/github.png" alt="github"></a></li>
+                    <li><a href="#"><img src="images/angellist.png" alt="angellist"></a></li>    
+                </ul>
+                <a class="mail" href="mariaamaliagomezmoro@gmail.com">mariaamaliagomezmoro@gmail.com</a>
+            </section>
+            <br>
+            <br>
+            <br>
+            <section class="LoremIpsum">
+            <div class="snapshot">
+                <img class="phoneimg" src="images/Snapshoot.png" alt="cover picture">
+                <img class="tabletimg" src="images/Snapshootbig.png" alt="">
+            </div>
+            <div class="wrap"><h3 class="keep">Keeping track of hundreds of components</h3>
+             <div class="button1"><button class="Live" type="button"></button>
+            <button class="Source" type="button"></button></div>
+            </div>
+            <div>
+            <ul class="language">
+                <li class="phone"><button type="button">Ruby on rails</button></li>
+                <li class="phone"><button type="button">css</button></li>
+                <li class="phone"><button type="button">JavaScript</button></li>
+                <li class="tablet"><button type="button">Codekit</button></li>
+                <li class="tablet"><button type="button">GitHub</button></li>
+                <li class="tablet"><button type="button">Bootstrap</button></li>
+                <li class="tablet"><button type="button">JavaScript</button></li>
+                <li class="tablet"><button type="button">Terminal</button></li>
+                <li class="tablet"><button type="button">Codepen</button></li>
+            </ul>
+            </div>
+            <div>
+            <p class="lorem">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea</p>
+            </div>
+            <div class="button">
+            <button class="Live1" type="button"></button>
+            <button class="Source1" type="button"></button>
+            </div>
+        </section>
```

Comment:
- [ ] Based on the former reviewer's request, kindly remove this section. If you look at the Figma template design, you will see that it ends at the footer. Kindly implement this change. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/172946010-6964592d-2ece-4e59-a4fe-face81627173.png)


### Figma Design

![image](https://user-images.githubusercontent.com/77106962/172945927-f79e92e1-509c-4e46-ad39-edaa6c83fe41.png)